app/src/main/res/layout/activity_campaign_details.xml

```diff
@@ -14,64 +14,136 @@
+            android:orientation="vertical"
+            android:paddingTop="50dp"
+            android:paddingLeft="@dimen/padding_medium"
+            android:paddingRight="@dimen/padding_medium"
+            android:paddingBottom="@dimen/padding_medium"
+            android:background="@color/web_orange"
+            android:layout_below="@+id/problemFact">
 
-        <org.dosomething.letsdothis.ui.views.typeface.CustomTextView
-            android:id="@+id/solutionSupport"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"/>
+            <org.dosomething.letsdothis.ui.views.typeface.CustomTextView
+                android:id="@+id/solutionCopy"
+                android:textAppearance="?android:textAppearanceLarge"
+                app:typeface="brandon_bold"
+                android:textSize="@dimen/text_33"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                tool:text="solution copy"/>
+
+            <org.dosomething.letsdothis.ui.views.typeface.CustomTextView
+                android:id="@+id/solutionSupport"
+                android:textAppearance="?android:textAppearanceLarge"
+                app:typeface="brandon_bold"
+                android:textSize="@dimen/text_33"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/solutionCopy"
+                tool:text="solution support"/>
+        </LinearLayout>
 
         <LinearLayout
-            android:orientation="horizontal"
-            android:id="@+id/action_buttons"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:layout_below="@id/solutionWrapper">
 
             <org.dosomething.letsdothis.ui.views.typeface.CustomButton
                 style="@style/SecondaryButton"
-                android:id="@+id/prove_share"
-                android:layout_weight="1"
-                android:layout_width="0dp"
+                android:id="@+id/prove_it"
+                android:textAppearance="?android:textAppearanceLarge"
                 app:typeface="brandon_bold"
+                android:textAllCaps="true"
+                android:textSize="@dimen/text_36"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
                 android:layout_margin="@dimen/padding_tiny"
-                android:text="@string/share_photo"
-                android:layout_height="wrap_content"/>
+                android:text="prove it"/>
 
             <org.dosomething.letsdothis.ui.views.typeface.CustomButton
-                android:id="@+id/invite"
+                style="@style/SecondaryButton"
```

Comment:
FIXME: This is the "Invite Friends" button, and it's style should be "PrimaryButton".